.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "libraries/standard/http/third_party/http_parser"]
 	path = libraries/standard/http/third_party/http_parser
 	url = https://github.com/nodejs/http-parser.git
+[submodule "libraries/3rdparty/CMock"]
+	path = libraries/3rdparty/CMock
+	url = https://github.com/ThrowTheSwitch/CMock
```

Comment:
do we always pull from master of CMock or is there a release tag that we are planning to use?

Reply:
If we put the release version in here, it has no effect on how gitsubmodules works and will not get automatically pulled. We need to go into the submodule and checkout the release we want and check that change in.

Reply:
AFR has it this way too but putting a release tag isn't a bad idea.

Reply:
Git stores the actual commit-id in the git database, you can retrieve it as explained here https://stackoverflow.com/questions/3983829/how-can-i-get-a-git-submodules-associated-commit-id-from-a-past-commit-in-the-p

Reply:
Ok. Thanks.